packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -83,6 +91,19 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
         ];
         const masks = ['Mask1', 'Mask2'];
         return html`
+            <slot
+                @slotchange=${() => {
+                    const labelFromSlot = getLabelFromSlot(
+                        this.label,
+                        this.slotHasContent,
+                        this.slotEl
+                    );
+                    if (labelFromSlot) {
+                        this.label = labelFromSlot;
+                    }
+                }}
```

Comment:
Anonymous methods in the render template will unbind and rebind every time the element is rendered, please move this to a class method.